src/schemata/definitions/workflow-template/action-objects/webhook.yml

```diff
@@ -33,6 +33,9 @@ additionalProperties:
                     type: string
                   access_key:
                     type: string
+              oidc:
+                type: object
+                additionalProperties: false
```

Comment:
this will only allow an empty object as a value, is that the intention?

Reply:
lets go with the following:

```
oneOf:
  - type: object
    additionalProperties: false
    properties:
      s3:
        type: object
        additionalProperties: false
        properties:
          secret_key:
            type: string
          access_key:
            type: string
  - type: string
    enum:
      - oidc
```

Reply:
```diff
-    enum:
-      - oidc
+ const: oidc
```

Reply:
Can someone also write up a description for all this and explain what is what, what it does, possibly add external links if appropriate, etc. We should start with the basics and spell out the `oidc` acronym, explain the use case, etc.

Reply:
oidc = OpenID Connect.. honestly, both is equally cryptic to someone who don't already know what it is.. :sweat_smile:

Reply:
I know what it means. My point is that there is ZERO context here. Are you suggesting that it's a reasonable expectation for all workflow authors to look at these TWO LINES and be like _"Ah, of course, this is how I use OpenID Connect to inject the active user session in workflows."_